man/rmdhunks/elevator-pitch.Rmd

```diff
@@ -0,0 +1,8 @@
+* **Setup vcr for your package with `vcr::use_vcr()`**
+* Tweak the configuration to protect your secrets
+* **Sprinkle your tests with `vcr::use_cassette()` to save HTTP interactions to disk in "cassettes" files**
+* If you want to test for package behavior when the API errors, edit the cassettes, or use [webmockr](htts://docs.ropensci.org/webmockr)
```

Comment:
> If you want to test for package behavior when the API errors

I think you're talking about error behavior of the API, like test that the package function handles e.g., an API error of 404 or 503, correct?